docs/content/latest/explore/multi-region-deployments/read-replicas-ysql.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Read replicas
+headerTitle: Read replicas
+linkTitle: Read replicas
+description: Read replicas
+menu:
```

Comment:
Could add the Beta tag/icon

Reply:
I'm not sure in this case. We usually add a beta tag when there's content.

Reply:
Let's see if somebody complains.